### DISCUSSION TITLE
Can somebody just post here a working client with this mod in it?

### DISCUSSION THREAD

**Discussion by magpie-o** [sultim-t/xash-rt#110] [General]
It can be good, yo

**Comment by TheInimitable** [comment]
No.

**Comment by roboboredom** [comment]


**Reply by magpie-o** [reply]
Yes, and I wish you the best too, kind stranger
If you will look at "Discussions", you will see that a lot of people cannot make this mod work. That is why I created this post.

**Reply by roboboredom** [reply]
Apologies. My comment was not in good faith and was too harsh.

**Comment by Implicit-D** [comment]
> It can be good, yo

Unfortunately, we can't provide a functional client, as that would technically be piracy. You have to provide the retail HL1 files and then modify them. To provide you with a client would be distributing the game for free, which is illegal. Sorry man. Someone might be able to set up a .bat file or something to automate the installation. Honestly though, it's incredibly easy with the included instructions, I'm not sure an automated means would actually be easier tbh.

**Reply by pirate** [reply]
Can you post a screenshot of it in the README? Curious what it looks like but dont have the time to do the whole setup. Seems like an epic project! <3
